go.mod

```diff
@@ -1,28 +1,31 @@
 module github.com/ONSdigital/dp-legacy-redirector
 
-go 1.19
+go 1.22
 
 require (
-	github.com/ONSdigital/dp-healthcheck v1.5.0
-	github.com/ONSdigital/go-ns v0.0.0-20210831102424-ebdecc20fe9e
-	github.com/ONSdigital/log.go/v2 v2.3.0
-	github.com/gorilla/mux v1.8.0
+	github.com/ONSdigital/dp-healthcheck v1.6.3
+	github.com/ONSdigital/dp-net/v2 v2.11.2
+	github.com/ONSdigital/log.go/v2 v2.4.3
+	github.com/gorilla/mux v1.8.1
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/smartystreets/goconvey v1.7.2
+	github.com/smartystreets/goconvey v1.8.1
 )
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
-	github.com/ONSdigital/dp-api-clients-go/v2 v2.187.0 // indirect
-	github.com/ONSdigital/dp-net v1.5.0 // indirect
-	github.com/ONSdigital/dp-net/v2 v2.6.0 // indirect
-	github.com/fatih/color v1.13.0 // indirect
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.260.0 // indirect
+	github.com/fatih/color v1.16.0 // indirect
+	github.com/go-logr/logr v1.4.1 // indirect
+	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/gopherjs/gopherjs v1.17.2 // indirect
 	github.com/hokaccha/go-prettyjson v0.0.0-20211117102719-0474bc63780f // indirect
 	github.com/jtolds/gls v4.20.0+incompatible // indirect
 	github.com/justinas/alice v1.2.0 // indirect
-	github.com/mattn/go-colorable v0.1.12 // indirect
-	github.com/mattn/go-isatty v0.0.16 // indirect
-	github.com/smartystreets/assertions v1.13.0 // indirect
-	golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab // indirect
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.20 // indirect
+	github.com/smarty/assertions v1.15.1 // indirect
+	go.opentelemetry.io/otel v1.24.0 // indirect
+	go.opentelemetry.io/otel/metric v1.24.0 // indirect
+	go.opentelemetry.io/otel/trace v1.24.0 // indirect
+	golang.org/x/net v0.20.0 // indirect
+	golang.org/x/sys v0.18.0 // indirect
```

Comment:
Did you run `go mod tidy` after bumping these?
I did, and there were many changes. 🤷

Reply:
Done... thanks

Reply:
Done